### ISSUE TITLE
Meeting with Tom Thomison (Creator of Holacracy) - 15th August

### ISSUE THREAD

**Issue by Pol-Lanski** [Giveth/Communication#65]
Lanski will be meeting Tom Thomison in Bangkok the 15th of August.

WRITE YOUR QUESTIONS BELOW if you have anything that you'd like to get some feedback from him.

- [ ] Review Roles Document that Kris and Lorelei are preparing
- [] What is the bare minimum for holacracy to work? What are the interdependencies between processes and systems for something to eliminate the inefficiencies that we are finding?

**Comment by krrisis** [comment]
- This is the roles sheet: https://docs.google.com/spreadsheets/d/10yFmRviEoeUCg8GfWizHBtkFqU9kgm_OhmIj9Lt-ONU/edit?usp=sharing

- This is the goals doc: https://docs.google.com/spreadsheets/d/10yFmRviEoeUCg8GfWizHBtkFqU9kgm_OhmIj9Lt-ONU/edit?usp=sharing

**Comment by krrisis** [comment]
Clear question that popped up: what do we do separately as circles, what do we do as a group? Personally I think the comms & governance circle could play very much an advisory role (but then they should -if possible always - be included in topics linked to the roles held by people in these respective circle. The social coding & dapp circles I personally see more as specialized circles. They do pick up both communication & governance-related roles but they do seek advice from their peers in the two advisory circles who have specific domains they govern (eg comms publishing space or governance the roles meeting). Thanks for the inspiration @Danibelle @vojtechsimetka @Pol-Lanski -- plz do add other Qs here, I didn't take well enough notes ;) (sorry)

**Comment by krrisis** [comment]
Should we pick one governance model such as holacracy and adopt it entirely & apply meticulously or can we just go on to make our 'own version' of it as Griff always says? Personally I really don't like that we don't have clear procedures, as it causes sooo many unneeded tensions and extra work for a few individuals (as they feel responsible & pick up the slack of others). More power to the roles would help a lot I think - so yeah, strong believer in the roles sheet, but as @vojtechsimetka said, indeed very important that we don't have generic roles but goal-oriented roles with detailed accountabilities. #hearhear

**Comment by Danibelle** [comment]
> what do we do separately as circles, what do we do as a group?
I think the light of transparency with our circle meetings and activities illuminates the path for managing this somewhat organically. 

1) we say what we are going to do, where those affected / with input can see it and comment, followed by 2) doing what we said, and 3) communicating that, allowing a bit of space for a feedback loop where anyone (eventually SOMEone who holds a role for tracking interdependencies) can 4) raise a flag, saying, Hey, this should probably go through another level of advisory engagement.

Then there's 5) determining whether a procedure, protocol, policy etc. may be needed to provide future guidance based on what we learned.

And where it felt we got to in the call, is that each circle has a role (or accountability within a role), for tuning in to the other circles, tracking what is being done, and identifying impacts with their own or anothers circle.. which feels good!

The suggestion about a 'Big Picture Person' (my analogy), who sees the larger whole of what EVERYONE is working on which has historically been that CEO-type role in a hierarchical org, that to me is sort of a program management aspect that will also develop itself in time as a role materializes that looks for efficiencies in the overall work flow between different projects within the Giveth Galaxy.
